src/engines/Cesium/common.ts

```diff
@@ -138,13 +139,18 @@ export const useIcon = ({
       ? Math.floor(img.height * imageSize)
       : Math.floor((w / img.width) * img.height);
 
-  const draw = useCallback(
-    (can: HTMLCanvasElement) =>
-      drawIcon(can, img, w, h, crop, shadow, shadowColor, shadowBlur, shadowOffsetX, shadowOffsetY),
-    [crop, h, img, shadow, shadowBlur, shadowColor, shadowOffsetX, shadowOffsetY, w],
+  const canvas = drawIcon(
```

Comment:
Could you use useMemo because this task should be heavy.

Reply:
Sure!